scripts/patches/item-base-roll-patch.mjs

```diff
@@ -48,10 +48,10 @@ export function patchItemBaseRoll() {
             chatMessageData = chatMessage.data;
         }
 
-        if (this.hasDamage && autoRollDamageWithOverride) {
-            // temporary until this can be added to core
-            const spellLevel = chatMessageData?.flags?.[MODULE_NAME]?.['spellLevel'] ?? this.data.data?.level;
+        // temporary until this can be added to core
+        const spellLevel = chatMessageData?.flags?.[MODULE_NAME]?.['spellLevel'] ?? this.data.data?.level;
 
+        if (this.hasDamage && autoRollDamageWithOverride) {
             const options = { spellLevel };
             if (Number.isNumeric(config?.spellLevel)) options.spellLevel = config.spellLevel;
```

Comment:
Looking again, these two lines should move out of this block as well. In case `options` is provided with an explicit spellLevel.

Reply:
Wasn't sure if it was specific to damage.
Anyway, done!